test/integration/pj-rehearse/expected.yaml

```diff
@@ -702,9 +702,9 @@
           - secret:
               name: cluster-secrets-test-profile
           - configMap:
-              name: rehearse-cluster-profile-test-profile-47224c86
+              name: rehearse-1234-cluster-profile-cluster-profile-test-profile
       - configMap:
-          name: rehearse-template-test-template-4b355dc3
+          name: prow-job-cluster-launch-src
```

Comment:
This CM not being changed is the consequence of the change in template substitution logic. This job uses a `test-template.yaml` from a `prow-job-cluster-launch-src` CM, but that CM is not handled at all by our config updater (see the [testing config](https://github.com/openshift/ci-tools/blob/921be048c697211820d3c5afbaec88d8e8273fe9/test/integration/pj-rehearse/master/core-services/prow/02_config/_plugins.yaml#L1-L8). Previously the rehearsal assumed the filenames match, therefore the file should be the same, but in production we should never have such case. New version of rehearse sees that this job does not use any of the config-updater-managed CMs, so it has no reason to do the substitution.